cockatrice/src/player.cpp

```diff
@@ -1104,13 +1104,14 @@ void Player::actCreateRelatedCard()
     QAction *action = static_cast<QAction *>(sender());
 
     // removes p/t from tokens (and leading space))
-    QStringList spaces = action->text().split(" ");
+    // Added split for "Token:" due to change in PR fixing #2317
+    QStringList spaces = action->text().split(tr("Token: "))[1].split(" ");
```

Comment:
I don't think that 6 additional characters are a very big deal.

It could be just "Create 3/3 Elephant" (same as the new card text btw) as well... I think it helps the user a lot and ui/ux wise it's better if the feature label is self-explaining.
While writing this, I even think it's the superior way to go.